### PR TITLE
CfN Redesign (March 2016)

### PR DESCRIPTION
**Pull request by bbenitez**
Cleaned up some files and removed Bootstrap. Structure remains practically the same; Changes are mostly aesthetic.

### CODE REVIEW

**Comment by beck410** [comment]
could you include a screenshot of the different pages that are affected?

**Comment by beck410** [comment]
could you include a screenshot of the different pages that are affected?

I'll take a look at this tonight

**Comment by bbenitez** [comment]
Sure thing! Here they are: Home, New Members, and Contact Form.


cfn_home


cfn_new-members


cfn_contact-form

**Comment by enlore** [comment]
+1

File: _includes/about.html [hunk 1]
```diff
@@ -1,20 +1,19 @@
...
-            </div>
+<div class="new-members">
+    <p>New To Code For Nashville? Visit our <a href="/new-members/">New Members page</a>.</p>
+</div>
+<section id="about">
+    <div class="about-cfn">
+        <h2>About</h2>
+        <p>Code for Nashville is Nashville's new <a href="http://www.codeforamerica.org/">Code for America Brigade</a>. We are volunteers who collaborate with Metro government to promote <a href="https://en.wikipedia.org/wiki/Open_data">open data</a> and build <a href="https://en.wikipedia.org/wiki/Civic_application">civic apps</a>.</p>
+        <p>We are a growing organization and need members with all types of skills.</p>
+        <p>The best way to become involved is to attend our next <a href="http://www.meetup.com/code-for-nashville/">hack night</a>. We meet monthly at the Sevier Park Community Center.</p>
```
Comment (beck410) [inline _includes/about.html]: the last sentence needs to be removed as we don't meet at sevier park anymore - i think linking to meetup is the best option for now while we are still structuring our meetups

File: _includes/team.html [hunk 2]
```diff
@@ -1,22 +1,12 @@
...
-                <a href="http://www.github.com/{{member.github}}" class="member-github-link">
-                    <img src="{{member.img}}" class="member-img" alt=""/>
-                    <h5 class="github-name" data-github="{{member.github}}">{{member.name}}</h5>
-                </a>
-            </div>
-          </div>
+        <li>
+            <h3>{{member.name}}</h3>
+            <p>{{member.title}}</p>
+            <a href="http://www.github.com/{{member.github}}">Github</a>
```
Comment (beck410) [inline _includes/team.html]: is there a reason to not include the images anymore? They were mainly there for people to know who to come and talk to at hack nights

File: new_members.html [hunk 3]
```diff
@@ -2,51 +2,71 @@
...
-</div>
+<div class="even">
+    <section class="modules">
+        <h2>Where to Find Us</h2>
+        <div>
+            <h3><a href="http://www.meetup.com/code-for-nashville/">Meetup</a></h3>
+            <p>This is where you can find dates for our monthly hack nights and leadership meetings. Each event is listed on meetup.com at least three weeks from the date. Here you can RSVP for an event, view what will be discussed each meeting and see who is attending.</p>
+        </div>
+        <div>
+            <h3><a href="https://groups.google.com/forum/#!forum/code-for-nashville">Google Groups</a></h3>
```
Comment (beck410) [inline new_members.html]: lets remove this link as we don't really use google groups anymore

**Comment by beck410** [comment]
so the new design is using a different logo than all our other marketing material - i really think we should use the same logo everywhere

**Comment by beck410** [comment]
its looking really good - just added a few comments :)

**Comment by bbenitez** [comment]
Submitted new commit with changes requested by Rebecca.

**Comment by beck410** [comment]
fantastic thanks for those changes :+1: 

looks good to merge once travis-ci passes

**Comment by bbenitez** [comment]
Merged Sonda's changes and resolved conflicts.

**Comment by bbenitez** [comment]
Fixed Travis CI build error. Test passed successfully.

**Comment by ebbeck** [comment]
nice! merge away :)

**Comment by bbenitez** [comment]
I have no write access to merge PRs, so I'll leave it to you guys. 👍

I appreciate all the feedback! Thank you and goodnight!

**Comment by ebbeck** [comment]
we appreciate all the effort you put into this - looks fantastic!

**Comment by SondaSengupta** [comment]
@bbenitez , just saw the redesign in production. These changes look absolutely amazing!

**Comment by neolytics** [comment]
I have to agree. Very nice work everyone.

**Pull request merged**